src/modules/authMapping/authMapping.module.ts

```diff
@@ -1,9 +1,10 @@
-import { Module } from '@nestjs/common';
+import { Global, Module } from '@nestjs/common';
 
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { AuthMapping } from './entity/authMapping.entity';
 import { AuthMappingService } from './services/authMapping.services';
 
+@Global()
```

Comment:
Can you think of any con for this practice?

_Why I did this?_
I wanted to create a single `SwaggerAuth` decorator that could be called without knowing any implementation logic. Since `SwaggerAuth` adds [TokenGuard](https://github.com/CyFy-Blockchain/auth/pull/15/files#diff-30a935a603aa616feb3a78e49e08523e4f0561621c945d80e2641631fac8af49), which uses `AuthMappingService` under the hood. This requires every module using `SwaggerAuth` to explicitly import `AuthMappingService` as well. 
By making `AuthMappingService` globally available, `SwaggerAuth` can be used across modules without requiring additional imports.

@MHA10

Reply:
I believe it's good to go. @MHassanTariq